Source/WebKit/UIProcess/Cocoa/UserMediaCaptureManagerProxy.h

```diff
@@ -100,7 +100,7 @@ class UserMediaCaptureManagerProxy : public IPC::MessageReceiver {
     using TakePhotoCallback = CompletionHandler<void(Expected<std::pair<Vector<uint8_t>, String>, String>&&)>;
     void takePhoto(WebCore::RealtimeMediaSourceIdentifier, WebCore::PhotoSettings&&, TakePhotoCallback&&);
 
-    using GetPhotoCapabilitiesCallback = CompletionHandler<void(WebCore::PhotoCapabilitiesOrError&&)>;
+    using GetPhotoCapabilitiesCallback = CompletionHandler<void(Expected<WebCore::PhotoCapabilities, String>&&)>;
```

Comment:
you can also use the `PhotoCapabilitiesNativePromise::Result` type too. works in the IDL definition too.

Reply:
I would need to use `WebCore::RealtimeMediaSource::PhotoCapabilitiesNativePromise::Result`, which I think makes it harder to understand so I will leave it as-is